packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-9/test.sh

```diff
@@ -12,4 +12,20 @@ test_val $(agd q vstorage children published.psm.IST -o json | jq -r '.children
 test_val $(agd q vstorage children published.psm.IST -o json | jq -r '.children | first') ${PSM_PAIR//IST./}
 
 # Gov params
-test_not_val "$(timeout 3 agoric follow -l :published.psm.${PSM_PAIR}.governance -o jsonlines | jq -r '.current.MintLimit.value.value')" "0" "PSM MintLimit non-zero"
\ No newline at end of file
+test_not_val "$(timeout 3 agoric follow -l :published.psm.${PSM_PAIR}.governance -o jsonlines | jq -r '.current.MintLimit.value.value')" "0" "PSM MintLimit non-zero"
+
+test_wallet_state() {
```

Comment:
We want to repeat this? To avoid cache-busting or something? Or just because we didn't get around to DRYing it out?

Reply:
Urf.  Job left undone while I was testing.  I'll DRY it now that CI has passed.